.travis.yml

```diff
@@ -60,44 +61,33 @@ before_install:
     - PROJECT_DIR=`pwd`
     - git --version
     - BOOST=$HOME/boost-local
-    - git clone -b $BRANCH_TO_TEST https://github.com/boostorg/boost.git $BOOST
+    - git clone -b $BRANCH_TO_TEST --depth 1 https://github.com/boostorg/boost.git $BOOST
     - cd $BOOST
-    - git submodule update --init --merge
+    - git submodule update --init --depth 1
     - ./bootstrap.sh
     - ./b2 headers
 
 script:
-    - cd $PROJECT_DIR/build
+    - cd $PROJECT_DIR
     - export BOOST_ROOT=$BOOST
     # `--coverage` flags required to generate coverage info for Coveralls
-    - ./build.sh --toolset=$CC "cxxflags=-std=$CXX_STANDARD -Wno-unused-local-typedefs -Wno-unused-function -Wno-deprecated-declarations --coverage" "linkflags=--coverage" -j3
+    - build/build.sh --toolset=$CC "cxxflags=-std=c++$CXX_STANDARD -Wno-unused-local-typedefs -Wno-unused-function -Wno-deprecated-declarations --coverage" "linkflags=--coverage" -j3
+    # CMake build
     - cd $BOOST && ./b2 --with-test --with-thread --with-chrono --with-system --with-atomic --with-date_time -a -j3 # Build required libs
     - mkdir $PROJECT_DIR/__build && cd $PROJECT_DIR/__build
-    - export CXXFLAGS="-std=$CXX_STANDARD"
-    - cmake .. -DCMAKE_BUILD_TYPE=Debug
+    - cmake .. -DCMAKE_BUILD_TYPE=Debug -DCMAKE_CXX_FLAGS="-std=c++$CXX_STANDARD"
```

Comment:
nit: not a huge fan of moving the `c++` prefix out of `CXX_STANDARD` there, but that’s no big deal

Reply:
This was actually part of a partial revert. Newer B2 support things like `cxxstd=14,17,20`. I wanted to use that but noticed that Boost 1.58 does not support that yet. I kept it so it can be changed once the minimum boost version tested is raised

Side note: There is boost-ci (on Github), which has scripts and CI templates for testing boost libs. This would make this much simpler. Check how different configs are added so many compilers and standards can be tested: https://github.com/boostorg/boost-ci/blob/3c8bd2be6d15184aebd8edb061512dd59eda0b96/.travis.yml#L177-L191

Reply:
Fair enough, thanks!